### PR TITLE
Delete game

### PR DESCRIPTION
**Pull request by gabrielmpinto**
Update custom domain commands to be clearer

### CODE REVIEW

File: src/cli/delete.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            return;
+        }
+    }
+
+    console.log("Deleting game...");
+    try {
+        const deleted = await playpassClient.deleteGame(config.game_id);
+        console.log(`${kleur.green("✔")} Deleted game ${deleted.name}`);
+    } catch (e: unknown) {
+        throw new Error(`${kleur.red("✘")} Failed to delete game: ${e}`);
```
Comment (aduros) [inline src/cli/delete.ts:39]:
```suggestion
        throw new Error(`Failed to delete game: ${e}`);
```

A red X is already prepended by the CLI's error handler.

File: src/cli/delete.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    }
+    return;
```
Comment (aduros) [inline src/cli/delete.ts:41]:
```suggestion
```

File: src/cli/playpass-client.ts [hunk 3]
```diff
@@ -109,6 +109,20 @@ export default class PlaypassClient {
...
+    public async deleteGame(gameId: string): Promise<Game> {
+        return axios.request({
+            method: "DELETE",
+            url: `${this.host}/api/v1/games/${gameId}`,
+            headers: {
+                "X-API-TOKEN": this.authToken
+            }
+        })
+            .then((res: AxiosResponse<Game>) => {
+                // PlaypassClient.validateResponse(res, "Failed to delete game.");
```
Comment (aduros) [inline src/cli/playpass-client.ts:121]: Do we not need to validate the response here?

**Review by aduros — CHANGES_REQUESTED** [review]

File: src/cli/delete.ts (hunk 2 above)
Comment (nhydock) [reply src/cli/delete.ts:41]: this just made me realize that we don't have a lint check on this repo.  We really should have one.

File: src/cli/delete.ts (hunk 1 above)
Comment (gabrielmpinto) [reply src/cli/delete.ts]: Fixed

File: src/cli/delete.ts (hunk 2 above)
Comment (gabrielmpinto) [reply src/cli/delete.ts]: Fixed

File: src/cli/playpass-client.ts (hunk 3 above)
Comment (gabrielmpinto) [reply src/cli/playpass-client.ts]: Fixed

**Review by aduros — APPROVED** [review]

**Pull request merged**